.github/workflows/sdk-jobs-pipelines-1h_automl_in_pipeline-automl-text-classification-multilabel-in-pipeline-automl-text-classification-multilabel-paper-categorization-in-pipeline.yml

```diff
@@ -0,0 +1,55 @@
+name: sdk-jobs-pipelines-1h_automl_in_pipeline-automl-text-classification-multilabel-in-pipeline-automl-text-classification-multilabel-paper-categorization-in-pipeline
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 */8 * * *"
+  pull_request:
+    branches:
+      - main
+      - sdk-preview
+      - pipeline/*
+      - may-sdk-preview
+      - ayushmishra/*  # To run workflow on current run
```

Comment:
We might not able to merge it this way? Could you rename the branch to pipeline/ayushmishra/update_automl_job_in_pipeline so that it is picked up?

Reply:
@skasturi sure. Will close this and create a new PR